src/components/templates/PartyMap/components/Map/MapRoom.scss

```diff
@@ -29,6 +29,18 @@ $label-visible: flex;
     }
   }
 
+  &.maproom--always-show-label-count {
+    .maproom__label {
+      display: $label-visible;
+    }
+  }
+
+  &.maproom--always-show-label-count {
+    .maproom__roomtitle {
+      display: $label-hidden;
+    }
+  }
```

Comment:
While I can see the approach you were going for here, I think there's a simpler way we can implement this feature (see other comments in this review)

Reply:
FYI, this pattern of CSS is called BEM (Block, Element, Modifier), which you can read more about at:

- http://getbem.com/naming/

In a majority of cases, we would stick to setting the 'modifier' (eg. `--always-show-label`) on the 'block' (eg. `.maproom`), as it seems you have followed here. Sometimes (if needed), we can also set it on the element itself (eg. `.maproom__title--foo`).